tests/mailmap/mailmap_helpers.h

```diff
@@ -0,0 +1,44 @@
+#include "mailmap.h"
+
+typedef struct mailmap_entry {
+	const char *real_name;
+	const char *real_email;
+	const char *replace_name;
+	const char *replace_email;
+} mailmap_entry;
```

Comment:
"mailmap_helpers.h" is a bit weird, as there are no helpers in here. It's more like data, so "mailmap_testdata.h" would fit better